src/app/tests/suites/commands/interaction_model/InteractionModel.h

```diff
@@ -155,6 +155,7 @@ class InteractionModelCommands
     CHIP_ERROR SendCommand(chip::DeviceProxy * device, chip::EndpointId endpointId, chip::ClusterId clusterId,
                            chip::CommandId commandId, const T & value,
                            const chip::Optional<uint16_t> & timedInteractionTimeoutMs = chip::NullOptional,
+                           const chip::Optional<uint16_t> & busyWaitForMs             = chip::NullOptional,
```

Comment:
we changed a command from 9 arguments to now take 10 arguments. 4 of those are optional<uint16_t>. This is probably not long term maintainable ..... any chance we can use some builder pattern and deprecate this particular method?

Reply:
@andy31415  I made https://github.com/project-chip/connectedhomeip/pull/24091 to address this issue. I will rebased this one on ToT once https://github.com/project-chip/connectedhomeip/pull/24091 has landed.